Modules/zlibmodule.c

```diff
@@ -1923,13 +1971,50 @@ zlib_crc32_impl(PyObject *module, Py_buffer *data, unsigned int value)
     return value;
 }
 
+/*[clinic input]
+zlib.crc32_combine -> unsigned_int
+
+    crc1: unsigned_int(bitwise=True)
+        CRC-32 check value for sequence A
+
+    crc2: unsigned_int(bitwise=True)
+        CRC-32 check value for sequence B
+
+    blen: object
+        Length of sequence B
+    /
+
+Combine two CRC-32 check values into one.
+
+Given a CRC-32 check value 'crc1' of a sequence A and a CRC-32 check
+value 'crc2' of a sequence B of length 'blen', the returned checksum
+is the CRC-32 check value of A and B concatenated.
+[clinic start generated code]*/
+
+static unsigned int
+zlib_crc32_combine_impl(PyObject *module, unsigned int crc1,
+                        unsigned int crc2, PyObject *blen)
```

Comment:
zlib uses `len2` instead of `blen` here, which I think makes more sense (and agrees with the .rst doc)